src/solutions/rode_solutions.jl

```diff
@@ -114,8 +114,15 @@ function calculate_solution_errors!(sol::AbstractRODESolution; fill_uanalytic =
     end
 
     if fill_uanalytic
-        for i in 1:length(sol)
-            push!(sol.u_analytic, f.analytic(sol.prob.u0, sol.prob.p, sol.t[i], sol.W[i]))
+        empty!(sol.u_analytic)
```

Comment:
when would this be non-empty?

Reply:
Not usually, but it might happen (it happened to me): issue #215